operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/kubernetes/CruKubernetesDependentResource.java

```diff
@@ -0,0 +1,16 @@
+package io.javaoperatorsdk.operator.processing.dependent.kubernetes;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.javaoperatorsdk.operator.processing.dependent.Creator;
+import io.javaoperatorsdk.operator.processing.dependent.Updater;
+
+/**
+ * Adaptor Class for standalone mode for resources that manages Create, Update and Delete
+ *
+ * @param <R> Managed resource
+ * @param <P> Primary Resource
+ */
+public abstract class CruKubernetesDependentResource<R extends HasMetadata, P extends HasMetadata>
```

Comment:
renamed.
Basically the delete is usually does not have to be implemented for the kubernetes resources, since those are cleaned up by the garbage collector, that is why no delete just CRU.